pcsx2/DEV9/AdapterUtils.cpp

```diff
@@ -334,8 +334,11 @@ std::vector<IP_Address> AdapterUtils::GetGateways(ifaddrs* adapter)
 		{
 			std::vector<std::string_view> split = StringUtil::SplitString(line, '\t', true);
 			std::string gatewayIPHex{split[2]};
-			int addressValue = std::stoi(gatewayIPHex, 0, 16);
-			//Skip device routes without valid NextHop IP address
+			// stoi assumes hex valuea are unsigned, but tries to store it in a signed int,
+			// this results in a std::out_of_range exception for addresses ending in a number > 128.
+			// We don't have a stoui for (unsigned int), so use stoul for (unsigned long).
```

Comment:
Stoui typo as well.

Reply:
I slightly tweaked that sentence to hopefully make the intent a bit clearer